presto-hive/src/main/java/com/facebook/presto/hive/RecordCursorProvider.java

```diff
@@ -24,9 +25,9 @@
 import java.util.Optional;
 import java.util.Properties;
 
-public interface HiveRecordCursorProvider
+public interface RecordCursorProvider
```

Comment:
I think it makes sense to continue call this `HiveRecordCursorProvider` given this interface is very specifically for Hive.